Resources/Prototypes/_ES/Entities/Markers/Spawners/Distributed/cargo.yml

```diff
@@ -0,0 +1,50 @@
+- type: entity
+  id: ESSpawnerRandomDistributedCargo
+  parent: ESMarkerBase
+  name: Cargo Crate Spawner
+  components:
+  - type: Sprite
+    layers:
+    - state: crate
```

Comment:
the crate icon is fairly generic so you should add a new sprite to the spawner rsi that says "CARGO" or something so that they're easier to distinguish. There's a ton of crate spawners and having it denoted as specifically for cargo would improve readability when mapping.